polygeist/lib/Dialect/Polygeist/Transforms/BareMemRefToLLVM.cpp

```diff
@@ -635,6 +650,22 @@ struct StoreMemRefOpLoweringOld : public MemAccessLowering {
     return success();
   }
 };
+
+struct ViewMemRefOpLoweringOld : public ConvertOpToLLVMPattern<memref::ViewOp> {
+  using ConvertOpToLLVMPattern<memref::ViewOp>::ConvertOpToLLVMPattern;
+
+  LogicalResult
+  matchAndRewrite(memref::ViewOp viewOp, OpAdaptor adaptor,
+                  ConversionPatternRewriter &rewriter) const override {
+    const auto loc = viewOp.getLoc();
+    auto gepPtr = rewriter.create<LLVM::GEPOp>(
+        loc, getTypeConverter()->convertType(viewOp.getSource().getType()),
+        adaptor.getSource(), adaptor.getByteShift());
+    rewriter.replaceOpWithNewOp<LLVM::BitcastOp>(
+        viewOp, getTypeConverter()->convertType(viewOp.getType()), gepPtr);
+    return success();
```

Comment:
Do something similar to the comment above: check valid input `memref` and check conversion works

Reply:
done.